pkg/system/system.go

```diff
@@ -394,6 +395,38 @@ func RunYaml(cmd *cobra.Command, args []string) {
 	util.Panic(p.PrintObj(sys, os.Stdout))
 }
 
+// RunSystemVersionsStatus runs a CLI command
+func RunSystemVersionsStatus(cmd *cobra.Command, args []string) {
+	log := util.Logger()
+	klient := util.KubeClient()
+
+	sysKey := client.ObjectKey{Namespace: options.Namespace, Name: options.SystemName}
+	r := NewReconciler(sysKey, klient, scheme.Scheme, nil)
+	util.KubeCheckQuiet(r.NooBaa)
+
+	noobaaImage := ""
+	noobaaDbImage := ""
+	noobaaOperatorImage := ""
+
+	if r.NooBaa.Spec.DBImage == nil {
```

Comment:
Why is the existence of spec.DBImage is the condition to all?? Doesn't make sense?
Also if the spec is empty, it doesn't mean that the installed versions are matching the CLI versions, so we should read the image versions from the pods (sts/deployment).

So I suggest this function will unconditionally read the actual images from the deploy/sts (if they even exist), and then also calculate the desired images based on the spec, and then print those images and add a warning if the desired and actual differ.

Reply:
Regarding the spec.DBImage you are correct :) 

Regarding the check of the images, not sure I fully understand, I will push a change with 2 more functions that checking the images and printing  a warning. Hopefully it answers this point.